omero/sysadmins/unix/server-centos7-ice36.rst

```diff
@@ -11,19 +11,21 @@ This guide describes how to install the **recommended** versions, not all
 the supported versions.
 This should be read in conjunction with :doc:`../version-requirements`.
 
+This guide does not describe how to install OMERO.web.
+To deploy OMERO.web **separately** from OMERO.server (recommended), please read
+:doc:`install-web/walkthrough/omeroweb-install-centos7-ice3.6` or
+to deploy with OMERO.server :doc:`install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6`
+
 These instructions assume your Linux distribution is configured with a UTF-8
 locale (this is normally the default).
 
 For convenience in this walkthrough the main OMERO configuration options have
 been defined as environment variables. When following this walkthrough you can
-either use your own values, or alternatively source the following file:
+either use your own values, or alternatively source :download:`settings.env <walkthrough/settings.env>`:
 
 .. literalinclude:: walkthrough/settings.env
    :start-after: Substitute
```

Comment:
This settings file includes ```OMERO_WEB_PORT=80``` which is not used anywhere in this page now.

Reply:
This is going to be trickier I guess since we have one file in omero-install that we use